### DISCUSSION TITLE
Issues Process (New)

### DISCUSSION THREAD

**Discussion by tedpatrick** [pyscript/pyscript#1158] [General]
We are working to make Issues more actionable and represent real bugs within PyScript. Prior to using Github discussions, PyScript issues could hold bugs, doc changes, upstream bugs, package requests, and feature requests. To address this, we are working to tag the existing issues as follows:

<img width="838" alt="Screen Shot 2023-02-10 at 3 21 54 PM" src="https://user-images.githubusercontent.com/35942272/218199902-4c1dc609-bfa3-4eda-b2c2-5c509096d7e8.png">

Today we start to tag issues and move items to the proper place within Github discussions. Ideally, the result will make contributing to PyScript far easier as our Issues all represent real bugs that we can actually fix and ship.

Feedback welcome!

Thanks,

Ted Patrick

**Comment by marimeireles** [comment]
My only point it renaming bug_fix to something more generic so it represents issues that aren't bugs (like docs).

**Reply by ntoll** [reply]
Why not just call it `FIX`..?

**Comment by ntoll** [comment]
LGTM - nothing jumps out as being confusing or weird... it all makes sense to me..! :-)

**Comment by FabioRosado** [comment]
Just a small suggestion, for package and feature requests instead of closing the issue and manually opening it into discussions, we could use the  `convert to discussion` button 😄 

<img width="214" alt="Screenshot 2023-02-10 at 11 29 25" src="https://user-images.githubusercontent.com/3131401/218082446-81481e05-f50d-4b90-bee0-b9132cf13d52.png">

**Reply by tedpatrick** [reply]
Ideal find.

**Comment by marimeireles** [comment]
ok
I've just tested what we were wondering about non contributors being able to create and interact in discussions and they can
so I guess we're all good 👍
I guess I'll add the flowchart to our contributing.md so people can udnerstand our triage process and new contributors can have guidelines etc 🙂

I guess another thing I just found is that it's possible to close an issue as won't fix in the Github interface:
<img width="385" alt="Screenshot 2023-02-10 at 11 52 13" src="https://user-images.githubusercontent.com/17600982/218185074-67d58264-2d58-4f9d-a7d1-815e2fa7e03e.png">
Another tool we could use :) instead of won't fix tag.

**Comment by tedpatrick** [comment]
Updated diagram above to:
<img width="838" alt="Screen Shot 2023-02-10 at 3 21 54 PM" src="https://user-images.githubusercontent.com/35942272/218200019-6b77545b-de13-4345-9848-8efe9504b17b.png">
